lib/citizens_advice_form_builder/elements/collections/check_boxes.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module CitizensAdviceFormBuilder
+  module Elements
+    module Collections
+      class CheckBoxes < Base
+        include ActionView::Helpers::FormOptionsHelper
+
+        def render
+          tag.div(class: "cads-form-field") do
+            tag.fieldset(class: "cads-form-field__content cads-form-group cads-form-group--checkbox") do
+              safe_join([legend_html, hint_html, checkboxes_html])
+            end
+          end
+        end
+
+        private
+
+        def items
+          @options[:collection].map do |item|
+            label = value_for_collection(item, text_method)
+            value = value_for_collection(item, value_method)
+            checked = Array(current_value).include?(value)
+
+            {
+              name: field_name,
```

Comment:
I think if we change the `name` here to `"#{field_name}[]"` then the form should work with the design system component like the radio buttons does and we wouldn't have to copy over the html.